drivers/tty/serdev/pibridge.c

```diff
@@ -380,6 +380,7 @@ int pibridge_req_send_io(u8 addr, u8 cmd, u8 *snd_buf, u16 snd_len)
 }
 EXPORT_SYMBOL(pibridge_req_send_io);
 
+// TODO: use a timeout here as in pibridge_req_gate_tmt()?
```

Comment:
Note that `pibridge_req_io()` calls `pibridge_recv()`, which uses the default timeout `PIBRIDGE_IO_TIMEOUT` (10 msec). So we do use a timeout here, it's just that the caller cannot modify it. Hence this comment probably isn't necessary.

Granted, for *sending*, `pibridge_req_io()` calls `pibridge_req_send_io()`, which uses `MAX_SCHEDULE_TIMEOUT`. So there's no timeout for sending, only receiving. But `pibridge_req_gate_tmt()` does the same.

Reply:
The point here is the inconsistency of using an internal timeout in one case but providing a parameter for the timeout in the other case. Why is the caller able to set a timeout for exchanging gate messages but not for exchanging io messages?